functions/go/render-helm-chart/README.md

```diff
@@ -86,34 +90,44 @@ helmGlobals:
+    includeCRDs: bool
+    skipTests: bool
+    values:
+      valuesFiles: []string
+      valuesInline: map[string]interface{}
+      valuesMerge: string
+
 ```
 
-| Field        |  Description | Example
-| -----------: |  ----------- | -----------
-`helmGlobals`  | Parameters applied to all Helm charts
-`helmCharts`   | An array of helm chart parameters
-`chartHome`    | A filepath to a directory of charts. The function will look for the chart in this local directory before attempting to pull the chart from a specified repo. Defaults to "tmp/charts". When run in a container, this path MUST have the prefix "tmp/". | tmp/charts
-`configHome`   | Defines a value that the function should pass to helm via the HELM_CONFIG_HOME environment variable. If omitted, {tmpDir}/helm is used, where {tmpDir} is some temporary directory created by the function for the benefit of helm. This option is not supported when running in a container. It is supported only in exec mode (e.g. with kustomize) | /tmp/helm/config
-`name`         | The name of the chart | minecraft
-`version`      | The version of the chart | 3.1.3
-`repo`         | A URL locating the chart on the internet | https://itzg.github.io/minecraft-server-charts
-`releaseName`  | Replaces RELEASE_NAME in the chart template output | test
-`namespace`    | Sets the target namespace for a release (`.Release.Namespace` in the template) | my-namespace
-`valuesInline` | Values to use instead of default values that accompany the chart |  global: <br> &emsp; enabled: false <br> tests: <br> &emsp; enabled: false  
-`valuesFile`   | valuesFile is a remote or local file path to a values file to use instead of the default values that accompanied the chart. The default values are in '{chartHome}/{name}/values.yaml', where `chartHome` and `name` are the parameters defined above. | Using a local values file: path/to/your/values.yaml <br> <br> Using a remote values file: https://raw.githubusercontent.com/config-sync-examples/helm-components/main/cert-manager-values.yaml
-`valuesMerge`  | ValuesMerge specifies how to treat ValuesInline with respect to Values. Legal values: 'merge', 'override' (default), 'replace'. | replace
-`includeCRDs`  | Specifies if Helm should also generate CustomResourceDefinitions. Defaults to false. | true
-
-The only required field is `name`.
+| Field           |  Description | Example
+| -----------:    |  ----------- | -----------
+`helmGlobals`     | Parameters applied to all Helm charts
+`helmCharts`      | An array of helm chart parameters
+`chartArgs`       | Arguments that describe the chart being rendered. 
+`templateOptions` | A collection of fields that map to flag options of `helm template`. 
+`chartHome`       | A filepath to a directory of charts. The function will look for the chart in this local directory before attempting to pull the chart from a specified repo. Defaults to "tmp/charts". When run in a container, this path MUST have the prefix "tmp/". | tmp/charts
```

Comment:
Same comment here. `tmp/` or `/tmp`?

Reply:
For `chartHome`, the default is `tmp/`. IIUC when run in a container, the code is running at the root so it becomes the same thing as `/tmp`. However for `chartHome` it is important to distinguish, because when run locally (i.e. `exec`) it will look for files in `tmp/charts` by default, not `/tmp/charts`. I have updated the containerized `valuesFiles` examples to use `/tmp` to be explicit, but `chartHome` is `tmp` and not `/tmp` intentionally.